### PR TITLE
Integrate api-check with msbuild

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by javiercn** [comment]
@pranavkm

File: build/shade/_api-check.shade [hunk 1]
```diff
@@ -19,16 +17,16 @@ default PROJECT_JSON="project.json"
             break;
         }
 
-        var projectJson = new FileInfo(Path.Combine(srcDirectory.FullName, PROJECT_JSON));
-        if (!projectJson.Exists)
+        var projectFile = new FileInfo(Path.Combine(srcDirectory.FullName, srcDirectory.Name + ".csproj"));
```
Comment (pranavkm) [inline build/shade/_api-check.shade:20]: @natemcmaster is this true everywhere?

File: build/shade/_api-check.shade [hunk 2]
```diff
@@ -82,6 +79,14 @@ functions @{
         var buildFolder = new DirectoryInfo(Path.Combine(directoryInfo.FullName, "bin", configuration));
         if (!buildFolder.Exists)
         {
+            Log.Info("No build folder found for " + Path.Combine(directoryInfo.FullName, "bin", configuration));
+            return;
+        }
+
+        var assetsJson = new FileInfo(Path.Combine(directoryInfo.FullName, "obj", "project.assets.json"));
+        if(!assetsJson.Exists)
```
Comment (pranavkm) [inline build/shade/_api-check.shade:87]: `if (!` (spacing)

**Review by pranavkm — APPROVED** [review]

File: build/shade/_api-check.shade (hunk 1 above)
Comment (natemcmaster) [reply build/shade/_api-check.shade:20]: Yes, AFAIK

**Review by natemcmaster — APPROVED** [review]
This is fine for now, but as we have discussed in person, let's add an issue to move this into Internal.AspNetCore.Sdk in the near future.

**Pull request closed without merging**